### DISCUSSION TITLE
Undefined property: Spatie\\Backup\\Events\\BackupHasFailed::$exception

### DISCUSSION THREAD

**Discussion by lukman-dev** [spatie/laravel-backup#1608] [Q&A]
hello, I'm using the package laravel backup version 7.0, 
I get an error when I try to run the command `Artisan::call('backup:run --only-db')` on my controller, when I try to run it via terminal with the command `php artisan backup: run --only-db`, everything works fine. but when using Artisan::call in my controller, then I get an error :

```
"message": "Undefined property: Spatie\\Backup\\Events\\BackupHasFailed::$exception",
    "exception": "ErrorException",
    "file": "/Users/...../Documents/..../..../vendor/spatie/laravel-backup/src/Notifications/Notifications/BackupHasFailedNotification.php",
    "line": 24 ....,
```

I'm using laravel version 8 with php 8.0, 
maybe someone can help me with this problem, I'm very grateful 🙏🙏

**Comment by erikn69** [comment]
> laravel backup version 7.0,

That version has no support, upgrade to the latest or feel free to make a PR for that branch
